index.bs

```diff
@@ -976,7 +1058,7 @@ following Web IDL.
 
     : <dfn dfn>JSON-serialized client data</dfn>
     :: This is the [=UTF-8 encoding=] of the result of calling the initial value of {{JSON/stringify|JSON.stringify}} on a
-        {{CollectedClientData}} dictionary. To avoid ambiguity, the {{ScopedCredentialInfo}} and {{AuthenticationAssertion}} structures
+        {{CollectedClientData}} dictionary. To avoid ambiguity, the {{AuthenticatorAttestationResponse}} and {{AuthenticatorAssertionResponse}} structures
         contain the actual serializations used by the client to generate them.
```

Comment:
I am thinking that the sentence:
> To avoid ambiguity, the {{AuthenticatorAttestationResponse}} and {{AuthenticatorAssertionResponse}} structures contain the actual serializations used by the client to generate them.

..is confusing and redundant since (per suggestions above) we explain in [[#iface-authenticatorattestationresponse]] and [[#iface-authenticatorassertionresponse]] the reason for and importance of conveying the clientDataJSON intact.  So I suggest deleting the above sentence.